collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -39,14 +40,25 @@
         'lines': [
             ['users', 'Users', 'absolute', 1, 1]
         ]
+    },
+    'mem': {
+        'options': [None, 'Minecraft Memory Usage', 'mb', 'spigotmc', 'spigotmc.mem', 'line'],
```

Comment:
```suggestion
        'options': [None, 'Minecraft Memory Usage', 'MiB', 'spigotmc', 'spigotmc.mem', 'line'],
```

Let's make it MiB. The dashboard will be able to [convert units](https://github.com/netdata/dashboard/blob/ba1512b03b3fb2b097169ea1c13b88c2d9b529cc/src/utils/units-conversion.ts#L114-L120).